client/src/App.js

```diff
@@ -1,5 +1,161 @@
+import { veggies } from "./fixtures/AllVeggies.js";
+import { fruities } from "./fixtures/AllFruities.js";
+import { salads } from "./fixtures/AllSalads.js";
+import months from "./fixtures/months.js";
+import PictureCard from "./components/Card.js";
+
+import { useState } from "react";
+
+import styled from "styled-components";
```

Comment:
```suggestion
import { useState } from "react";
import styled from "styled-components";

import { veggies } from "./fixtures/AllVeggies.js";
import { fruities } from "./fixtures/AllFruities.js";
import { salads } from "./fixtures/AllSalads.js";
import months from "./fixtures/months.js";
import PictureCard from "./components/Card.js";
```

Imports should be structured 'outside in': First external libraries an then your own components